etl/steps/data/garden/happiness/2023-03-20/happiness.meta.yml

```diff
@@ -28,20 +44,10 @@ tables:
   happiness:
     title: World Happiness Report (2023)
     variables:
-      ladder_score:
+      cantril_ladder_score:
         title: Cantril ladder score
         description: Average of survey responses to the 'Cantril Ladder' question in the Gallup World Poll. The survey question asks respondents to think of a ladder, with the best possible life for them being a 10, and the worst possible life being a 0.
         short_unit: ''
         unit: ''
-      standard_error_of_ladder_score:
-        title: Standard error of Cantril ladder score
-        short_unit: ''
-        unit: ''
-      upperwhisker:
-        title: 97.5% confidence interval
-        short_unit: ''
-        unit: ''
-      lowerwhisker:
-        title: 2.5% confidence interval
-        short_unit: ''
-        unit: ''
+        display:
+          numDecimalPlaces: 2
```

Comment:
Is there a particular reason to use `numDecimalPlaces=2`? The general rule is to use `numDecimalPlaces=1`, unless more precision is needed (e.g. possible values range from 0 to 1).